providers/dnsmadeeasy/dnsMadeEasyProvider.go

```diff
@@ -177,8 +177,8 @@ func (api *dnsMadeEasyProvider) GetDomainCorrections(dc *models.DomainConfig) ([
 	return corrections, nil
 }
 
-// EnsureDomainExists returns an error if domain doesn't exist.
-func (api *dnsMadeEasyProvider) EnsureDomainExists(domainName string) error {
+// EnsureZoneExists returns an error if domain doesn't exist.
+func (api *dnsMadeEasyProvider) EnsureZoneExists(domainName string) error {
```

Comment:
domainName should be zoneName.

If we're going to make this change, we might as well change the "domain string" parameter in most of these other functions. I wouldn't change much more (like the api.domainExists() call below)

Reply:
done